source/fundamentals/crud/read-write-pref.txt

```diff
@@ -0,0 +1,235 @@
+
+   * - ``journal`` *(optional)*
+     - | Requests acknowledgment that the write operation has been written to the on-disk journal 
+       | **Type:** boolean
+
+Example: Set the Write Concern for a Single Write Operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This code uses custom ``WriteConcern`` settings while creating new a document:
+
+.. code-block:: js
+
+   myDB.myCollection.insertOne( 
+      { name: "anotherDocumentName" }, 
+      { writeConcern: 
+        { w: 2, wtimeoutMS: 5000 } 
+      } 
+   );
+
+Example: Create and Use a New Write Concern using the fromOptions and apply Methods
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+   
+This code use the ``fromOptions`` method to construct a ``WriteConcern`` from the options of an
+existing database, ``myDB``. Note that ``myDB`` could be replaced with any entity that
+includes a write concern option.
+
+.. code-block:: js
+
+   const newWriteConcern = WriteConcern.fromOptions(myDB);
+   const myDoc = { name: "New Document" };
+   WriteConcern.apply(myDoc,newWriteConcern);
+
+Read Concern
+------------
+
+The read concern specifies the following behaviors:
+
+- Level of :manual:`consistency </core/causal-consistency-read-write-concerns/>` across replica sets
+
+- `Isolation
+  <https://www.mongodb.com/docs/current/core/read-isolation-consistency-recency/>`__
+  guarantees maintained during a query 
+
+Lower consistency requirements may reduce latency. The default read concern level is
+``local``. This means that the client returns the data from the replica set member that the
+client is connected to, with no guarantee that the data has been written to all replica set members.
+
+For more information about read concerns or read concern levels, see 
+:manual:`Read Concern </reference/read-concern/>` in the Server manual. For more detail on
+the ``ReadConcern`` type and definitions of the read concern levels, see the `ReadConcern <{+api+}/classes/ReadConcern.html>`__ in
+the API documentation.
+
+Example: Set the Read Concern Level of an Aggregation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This code sets the read concern level of an an aggregation to ``"majority"``:
+
+.. code-block:: js
+
+   const pipeline = [
+      {"$match": {
+        category: "KITCHENWARE",
+      }},
+      {"$unset": [
+        "_id",
+        "category",
+      ]}
+    ]; 
+   
+   result = await myDB.collection("mycollection")
+      .aggregate(
+        pipeline, 
+        { readConcern: 
+          { level: "available" } 
+        } 
+      );
+
+For more information about aggregates, see the :ref:`nodejs-aggregation` page.
+   
+Example: Change the Read Concern of a Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This code changes the read concern level of a database to ``"local"``:
+
+.. code-block:: js
+
+   const options = { readConcern: { level: "local" } };
+   const myDB = client.db("db-name", options);
```

Comment:
Suggestion:

Just want to make sure that the [original suggestion](https://github.com/mongodb/docs-node/pull/819#discussion_r1401127153) was communicated clearly.

While both "db_name" and "db-name" are valid, I still think it could be good to use "mydb" or something similar (e.g. "myDB" used as an example on this manual page on Databases https://www.mongodb.com/docs/manual/core/databases-and-collections/#databases) since it would be consistent with the pattern used earlier in this doc for collection ("mycollection").


```suggestion
   const myDB = client.db("mydb", options);
```